pandas/_libs/arrays.pyi

```diff
@@ -26,7 +26,7 @@ class NDArrayBacked:
     def size(self) -> int: ...
     @property
     def nbytes(self) -> int: ...
-    def copy(self): ...
+    def copy(self, order: str = "C"): ...
```

Comment:
any reason this is `"C"` instead of `...`?

Reply:
In a PYI file, you don't specify default values.  Defaults would only go in the declaration of the function.  So it should be `order: str = ...` .  That means the `order` argument is optional.